lightmdb/application.py

```diff
@@ -26,7 +26,7 @@
 
 DEFAULT_APP_NAME = 'lightmdb'
 DEFAULT_APP_SECRET = 'Secret@LightMDB'
-DEFAULT_DSN = "user='vagrant' password='vagrant' host='localhost' port=54321 dbname='itucsdb'"
+DEFAULT_DSN = "user='vagrant' password='vagrant' host='localhost' port=5432 dbname='itucsdb'"
```

Comment:
biz de hocanın neden 54321 kullandığını anlamadık zaten, ama ona hiç dokunmadık. Lokalde kullanmak için en kolay yöntem postgresql kurmak ve daha sonrasında local_settings.py dosyasına kendi ayarlarını vermek. DEBUG=True yazmak gereke biliyor mesela, DSN-de de lokaldeki postgresql ayarlarını vere biliyorsun (readthedocs-da yazıldığı gibi).